Makefile

```diff
@@ -246,3 +246,13 @@ update_codegen:
 
 # Dummy force target for pre-build steps
 .PHONY: .FORCE
+
+#currently only for stm32
+#will need different bash window to run the python
+#to send CAN message write
+#can_message.send_message(1) to send can message
+.PHONY: babydriver
+babydriver:
+	@make program PROJECT=baby_driver
+	@python3
+	@from projects.baby_driver.scripts import can_message
```

Comment:
Once this solidifies, you might wanna look into not forcing 2 windows. You can run multiple things in parallel in the same command using [this](https://stackoverflow.com/a/9220818), you can still have it as multiple lines in the make file using [this](https://stackoverflow.com/a/10121575). 

The second command should still have stdin readable from the commandline, so you'd want to use 2 commands if there's stdin for both. If not and you're worried about stdout being cluttered by both, you can prefix the output using something like [this](https://unix.stackexchange.com/a/440438) to make things a bit easier on the eyes

Reply:
Oh right now, it doesn't need 2 windows. I'm saying we need 2 windows on x86 because 1 will show the output from the C project, and the other will be needed to run the python scripts. I'm not sure it's a parallel process issue, but I could be misinterpreting what you mean. If you want, we can set a time to talk about it to clarify things.